cs251tk/student/markdownify/markdownify.py

```diff
@@ -13,9 +13,10 @@ def markdownify(spec_id, *, username, spec, basedir, debug, interact, student, w
         first_submit = ''
 
         if not ci:
+            first_date = check_dates(spec_id, username, spec, basedir)
             first_submit = "First Submission for {}: {}".format(
```

Comment:
Does your formatter complain if you move the `.format` arguments onto this line?

Reply:
That seems to work.

Reply:
Can you please make this string formatting one line? `"".format(spec_id, first_date)`

Reply:
Ah. Sure.